scene/gui/color_picker.cpp

```diff
@@ -79,6 +393,30 @@ void ColorPicker::_notification(int p_what) {
 				labels[i]->set_custom_minimum_size(Size2(get_theme_constant(SNAME("label_width")), 0));
 				set_offset((Side)i, get_offset((Side)i) + get_theme_constant(SNAME("margin")));
 			}
+			alpha_label->set_custom_minimum_size(Size2(get_theme_constant(SNAME("label_width")), 0));
+			set_offset((Side)0, get_offset((Side)0) + get_theme_constant(SNAME("margin")));
+
+			Ref<StyleBoxEmpty> style_box_empty(memnew(StyleBoxEmpty));
+
+			for (int i = 0; i < 4; i++) {
```

Comment:
Hard coded, suggest using `current_sliders_count` instead of `4`

Reply:
In this case, it can be `4`, and there is no need to write the following `alpha_scroll` separately.

Reply:
`alpha_scroll` is actually separate from the 4 sliders.
The 4th slider is unused for now. It's potentially useful for CMYK or other color mode with more sliders (if there is any...). Although maybe it could be removed for now 🤔 This would be trivial if 4 was a constant not hard-coded.

Reply:
So it's better to use `current_sliders_count`, which is reset according to the mode in `ColorPicker::_update_controls`.

Reply:
Should I define a constant with 3 as its value? we will change it to 4 in future if needed.

Reply:
> `alpha_scroll` is actually separate from the 4 sliders. The 4th slider is unused for now. It's potentially useful for CMYK or other color mode with more sliders (if there is any...). Although maybe it could be removed for now 🤔 This would be trivial if 4 was a constant not hard-coded.

What if element 0 was always used for the alpha slider, and then all the code duplication with the other sliders and the alpha slider wouldn't be necessary? Would that work?